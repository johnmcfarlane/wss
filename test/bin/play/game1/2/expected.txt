 42  4,14 - FOH
 34  4,4  - JABIA
 33  5,14 - OF
 31  4,14 - HOI
 29  5,14 - OH
 27  3,14 - AHO
 26 15,8  | AHI
 25 14,4  | FATH
 25 15,9  | HA
 25 15,9  | HI
 23 15,8  | AH
 23 15,8  | OH
 23  3,10 | OHIA
 22  3,10 | AHI
 22  3,10 | AHO
 22  6,14 - FI
 21  3,10 | AH
 21 12,4  - FA
 21  4,14 - HO
 21  3,10 | OH
 21  5,14 - OI
 20  2,2  | FAH
 20  2,2  | FOH
 19  3,11 | HAF
 19  6,14 - IF
 19  6,14 - OF
 18 14,4  | FAT
 18 14,4  | FOT
 18 12,4  - HA
 18  6,14 - HI
 18  6,14 - HO
 18 15,11 | IDAHO
 18  2,2  | QI
 16  2,3  | AH
 16  6,12 | FOH
 16 15,10 | HAD
 16  3,11 | HAO
 16  3,11 | HIA
 16 15,10 | HID
 16  3,11 | HOI
 16  9,14 | IF
 16  9,14 | OF
 16  2,3  | OH
 16  6,14 - OH
 16 10,4  - OHIA
 16  2,13 | OHO
 16 12,14 - QI
 15  5,11 | FIZ
 15  3,11 | HA
 15 12,4  | HAO
 15  3,11 | HI
 15 11,4  - HIA
 15  3,11 | HO
 15  3,14 - IF
 15  3,14 - OF
 15  3,12 | QAID
 14  1,14 - AHA
 14  1,14 - AHI
 14  1,14 - AHO
 14 13,4  - AHU
 14  7,13 | OAF
 14  7,13 | OOF
 14  1,5  - RHA
 14  1,13 - WHF
 13  3,14 - AH
 13  9,14 | AH
 13 14,9  - CH
 13  3,14 - OH
 13  9,14 | OH
 13  5,13 | ZOL
 12  2,1  | AHI
 12  2,1  | AHO
 12 15,10 | AID
 12  5,9  - AMOR
 12  5,8  | DA
 12  2,2  | FA
 12 11,4  | FAVI
 12  2,2  | FI
 12 12,13 - FOD
 12  6,12 | FOO
 12  2,1  | HAO
 12  2,1  | HIA
 12  2,1  | HOI
 12 12,12 | IF
 12  6,12 | OOF
 12  5,3  - VAI
 11  3,13 | AFD
 11 15,12 | DAHI
 11  3,5  | FLO
 11  2,2  | HA
 11  2,2  | HI
 11  2,2  | HO
 11  7,13 | OOH
 11  1,3  - PHI
 11  1,3  - PHO
 11  1,13 - WHA
 11  1,13 - WHO
 10 15,12 | DAH
 10 12,14 - FA
 10 13,3  - FAC
 10 12,14 - FI
 10  6,12 | HOI
 10  6,12 | HOO
 10  6,14 - IO
 10 10,14 - NAIO
 10 10,14 - NAOI
 10  6,13 | OF
 10  6,14 - OI
 10  6,12 | OOH
 10  1,13 - WAF
  9 15,11 | AD
  9  2,2  | AI
  9 12,14 - HA
  9 11,4  | HAV
  9 12,14 - HI
  9 13,3  - HIC
  9 12,14 - HO
  9 13,3  - HOC
  9 15,11 | ID
  9  2,2  | IO
  9  3,7  - IO
  9  3,14 - IO
  9 14,11 - KA
  9 14,11 - KI
  9  2,2  | OI
  9  1,3  - PAH
  9  1,3  - POH
  9  2,9  - RAH
  9 13,12 | TOHO
  9  1,13 - WAH
  8 13,4  | AGLEE
  8  1,14 - AH
  8 14,5  | AT
  8 11,5  | AVO
  8 15,12 | DAO
  8 13,7  - EH
  8  3,13 | FAD
  8  3,13 | FID
  8  3,13 | FOD
  8 13,5  - GAU
  8  2,14 | HO
  8  9,14 - INFO
  8  1,11 - ISH
  8 11,5  | IVA
  8  4,6  | LOAF
  8 11,14 | OF
  8  7,13 | OFO
  8  6,13 | OH
  8 11,5  | OVA
  8 14,6  | TH
  7 11,14 | AH
  7  9,14 | AI
  7 12,14 - AI
  7 15,12 | DA
  7  3,13 | HAD
  7  3,13 | HID
  7  3,13 | HOD
  7  9,14 - INF
  7  9,14 | IO
  7 12,14 - IO
  7  6,9  - MORA
  7  6,9  - MORO
  7 10,14 - NAF
  7 13,13 - ODA
  7 11,14 | OH
  7  7,13 | OHO
  7  9,14 | OI
  7 12,14 - OI
  7  1,3  - PIA
  7  1,3  - POI
  7  2,9  - RHA
  6  1,10 - ABA
  6  1,10 - ABO
  6 14,4  - FU
  6 10,14 - NOH
  6  1,3  - PA
  6  1,3  - PI
  6  1,3  - PO
  6  1,13 - WO
  5  4,8  | AAH
  5  4,8  | AHA
  5  4,8  | AHI
  5  4,8  | AHO
  5 11,14 | AI
  5  3,13 | AID
  5  4,14 | FE
  5  3,5  | ILA
  5  9,14 - INIA
  5  3,13 | OAD
  5  7,13 | OF
  5 11,14 | OI
  5  3,5  | OLA
  5  2,9  - RIA
  5  2,9  - ROI
  5  1,4  - UH
  4  3,14 | AD
  4  4,8  | AH
  4 14,10 - AH
  4  1,14 - AI
  4  3,5  | AL
  4  9,14 - ANI
  4 13,7  - EO
  4  4,14 | HE
  4  6,12 | HO
  4  3,14 | ID
  4  2,14 | IO
  4  4,6  | LOA
  4 10,14 - NOA
  4  3,14 | OD
  4  7,13 | OH
  4  6,13 | OI
  4  9,14 - ONA
  4  9,14 - ONI
  4  6,13 | OO
  4 14,6  | TO
  4 13,12 | TOA
  4 13,12 | TOI
  4 13,12 | TOO
  3  9,14 - AN
  3  9,14 - IN
  3  3,6  | LA
  3  3,6  | LI
  3  3,6  | LO
  3 10,14 - NA
  3 10,14 - NO
  3  9,14 - ON
  2  4,8  | AA
  2 14,10 - AA
  2  4,14 | AE
  2  4,8  | AI
  2 14,10 - AI
  2  6,12 | IO
  2  4,14 | OE
  2  7,13 | OI
  2  6,12 | OO
  2  7,13 | OO
