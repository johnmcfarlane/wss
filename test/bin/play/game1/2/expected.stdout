11,3:FA=21
5,13:IF=19
11,3:HA=18
5,13:HI=18
1,1:QI=18
1,2:AH=16
5,11:FOH=16
14,9:HAD=16
14,9:HID=16
8,13:IF=16
8,13:OF=16
9,3:OHIA=16
1,12:OHO=16
4,10:FIZ=15
11,3:HAO=15
2,13:IF=15
2,13:OF=15
2,11:QAID=15
0,13:AHA=14
0,13:AHI=14
6,12:OAF=14
2,13:AH=13
8,13:AH=13
2,13:OH=13
8,13:OH=13
1,0:AHI=12
14,9:AID=12
1,1:FA=12
1,1:HA=11
1,1:HI=11
6,12:OOH=11
0,2:PHI=11
0,12:WHA=11
0,12:WHO=11
14,11:DAH=10
5,12:OF=10
5,11:OOH=10
14,10:AD=9
1,1:AI=9
12,2:HIC=9
11,13:HO=9
14,10:ID=9
13,10:KA=9
13,10:KI=9
1,1:OI=9
0,2:PAH=9
0,2:POH=9
1,8:RAH=9
12,3:AGLEE=8
0,13:AH=8
10,4:AVO=8
2,12:FAD=8
2,12:FID=8
1,13:HO=8
8,13:INFO=8
3,5:LOAF=8
10,13:OF=8
5,12:OH=8
10,4:OVA=8
10,13:AH=7
8,13:AI=7
2,12:HAD=7
2,12:HID=7
2,12:HOD=7
5,8:MORA=7
10,13:OH=7
6,12:OHO=7
8,13:OI=7
0,2:PIA=7
0,2:POI=7
0,9:ABA=6
0,9:ABO=6
9,13:NAH=6
9,13:NOH=6
0,2:PA=6
0,2:PI=6
0,12:WO=6
3,7:AAH=5
3,7:AHA=5
3,7:AHI=5
10,13:AI=5
2,12:AID=5
3,13:FE=5
8,13:INIA=5
6,12:OF=5
10,13:OI=5
1,8:RAI=5
1,8:RIA=5
0,3:UH=5
2,13:AD=4
13,9:AH=4
3,7:AH=4
2,4:AL=4
8,13:ANI=4
3,13:HE=4
5,11:HO=4
2,13:ID=4
2,13:OD=4
6,12:OH=4
5,12:OI=4
12,11:TOO=4
8,13:AN=3
8,13:IN=3
2,5:LA=3
2,5:LI=3
2,5:LO=3
9,13:NA=3
9,13:NO=3
8,13:ON=3
13,9:AA=2
3,7:AA=2
3,13:AE=2
13,9:AI=2
3,7:AI=2
3,13:OE=2
6,12:OI=2
